Makefile-os

```diff
@@ -2,18 +2,6 @@
 # Our makefile makes use of docker compose commands. Our config files rely on environment variables
 # both for passing configuration to the containers as well as configuring the compose file itself.
 # Variables referenced in docker-compose*.yml should be read from .env, exported and saved in .env
-
-define ENV_VAR
```

Comment:
This logic was already very hard to follow, and now with the requirement to additionally prefix the DOCKER_VERSION value, it makes a lot of sense to move this to a python script.